nemo/collections/asr/parts/utils/audio_utils.py

```diff
@@ -45,7 +45,7 @@ def get_samples(audio_file: str, target_sr: int = 16000):
         samples = f.read()
         if sample_rate != target_sr:
             samples = librosa.core.resample(samples, orig_sr=sample_rate, target_sr=target_sr)
-        samples = samples.transpose()
+        samples = samples.transpose().astype('float32')
```

Comment:
Does it work with this to avoid using more memory?
samples = f.read(dtype="float32')

If not, let's use this one to transpose smaller tensor:
samples.astype('float32').transpose()

Reply:
made f.read(dtype=dtype)
and moved dtype to have "float32" as a default argument.